be/src/exec/pipeline/aggregate/aggregate_distinct_streaming_sink_operator.cpp

```diff
@@ -78,7 +80,6 @@ Status AggregateDistinctStreamingSinkOperator::_push_chunk_by_force_preaggregati
 
     COUNTER_SET(_aggregator->hash_table_size(), (int64_t)_aggregator->hash_set_variant().size());
 
-    _mem_tracker->set(_aggregator->hash_set_variant().reserved_memory_usage(_aggregator->mem_pool()));
```

Comment:
why delete `_mem_tracker`? after deleted it, can the fragment's memory tracker know the update of memory usage in time to avoid OOM?

Reply:
The original `_mem_tracker` is only for recording the memory used by hash table. But aggregator can take care of it withouth relating a MemTracker outside of itself. 

After this pr, we can get memory usage of hash map/set by calling `Aggregator::hash_map_memory_usage` or `Aggregator::hash_set_memory_usage`

Reply:
ok